packages/ui/src/ui/store/actions/navigation/index.js

```diff
@@ -39,7 +39,7 @@ import {isSupportedEffectiveExpiration} from '../../../store/selectors/thor/supp
 export function updateView(settings = {}) {
     return (dispatch, getState) => {
         const state = getState();
-        const currentPath = getPath(state);
+        const currentPath = decodeURIComponent(getPath(state));
```

Comment:
It looks like an unnecessary change. Do we really need it?

Reply:
yes, otherwise the links will not work correctly